wftests/warrior_tests/testcases/selenium_func_tests/generic_tests/tc_selenium_stale_element_exception_1.xml

```diff
@@ -34,7 +34,7 @@
                 <step TS= '3' Driver='selenium_driver' Keyword='get_element_by_link_text' >
                     <Arguments>
                         <argument name="system_name" value="search_element"/>
-                        <argument name="link_text" value="Log In"/>
+                        <argument name="link_text" value="Log in"/>
```

Comment:
Address the issue with a separate ticket.

Reply:
Hi @santhosh-warrior 
I have created a ticket(WAR-2200) for the same and fixed it.